pkNX.Structures/Dumpers/PersonalDumper.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace pkNX.Structures;
 
+/* Now unused
```

Comment:
If it's unused, just delete it. That's what we have version control for

Reply:
Thanks!
Originally I downloaded an old version of project without the evolutions issue to use, but it still had this bug. I actually started editing that, so I basically mimicked the changes I made in the old one.
I will look at the function (I may have just forgot to copy my edits, or else I didn't think about its use).
I will also delete the old function I commented out. You make a valid point. I should have already done that, and probably just mentioned it in my post. Now that I think about it, GitHub would display that I deleted it anyways.
Regardless, I will get to correcting things. Appreciate you taking a look and catching that stuff!